nodejs/__test__/table.test.ts

```diff
@@ -448,6 +455,43 @@ describe("When creating an index", () => {
   });
   afterEach(() => tmpDir.removeCallback());
 
+  it('tests "offset" for "vectorSearch"', async () => {
+    // to trigger an error, change the default limit to 1 on the line below
+    const vectorSearch = (offset: number, limit = 1) =>
+      tbl
+        .vectorSearch(queryVec)
+        .limit(limit)
+        .offset(offset)
+        .select("id")
+        .toArray();
+    const rowsFirstTwo = await vectorSearch(0, 2);
+    expect(rowsFirstTwo.length).toBe(2);
+    const rowsFirst = await vectorSearch(0);
+    expect(rowsFirst.length).toBe(1);
+    const rowsSecond = await vectorSearch(1);
+    expect(rowsSecond.length).toBe(1);
+    expect([rowsFirst[0].id, rowsSecond[0]?.id]).toEqual(
+      rowsFirstTwo.map((v) => v.id),
+    );
+  });
+
+  it("runs full text search ", async () => {
+    const pathDb = path.join(tmpDir.name, ".lancedb");
+    const db = await connect(pathDb);
+    const schema = new arrow.Schema([
+      new arrow.Field("text", new arrow.Utf8()),
+    ]);
+    const table = await db.createEmptyTable("full-text-search", schema, {
```

Comment:
if a table is created using `createEmptyTable`, then `fts` search won't work afterwords.
- returns empty array, when should return 2 rows

Reply:
Hm does the index exist on the table? If there are no rows to index, it might not.

I would create the index after inserting data instead.

Reply:
The .listIndices() a couple of lines below returns an index

Reply:
If an index doesn’t exist, LanceDB returns an error that you must create an inverted index